scripts/third_party_size_check.py

```diff
@@ -21,23 +21,32 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
+with open("./app.yaml", 'r') as app_yaml:
+    try:
+        app_yaml_dict=yaml.safe_load(app_yaml)
+    except yaml.YAMLError as exc:
+        print(exc)
+skip_files = app_yaml_dict.get("skip_files")
+
+skip_files = [os.getcwd() + '/' + skip_files_dir  for skip_files_dir in skip_files]
 
 def _check_size_in_dir(dir_path):
     """Recursive method that checks the number of files inside the given
     directory.
-
```

Comment:
Why this change (and the below)? Whenever you're working on a PR, go to the "Files changed" section and see the diff. Ensure that it contains only relevant changes.

Reply:
Yes Fixed PTAL Thanks!